doc/tutorials/increase-uos-disk-size.rst

```diff
@@ -0,0 +1,51 @@
+.. _Increase UOS disk size:
+
+Increasing the User OS disk size
+################################
+
+This document builds on the :ref:`getting_started` and assumes you already have
+a system with ACRN installed and running correctly. The size of the pre-built
+Clear Linux User OS (UOS) virtual disk is typically of 8GB. This may not be
+sufficient and this guide explains in a few steps how to increase the size of
+that virtual disk.
+
+This document is largely inspired from Clear Linux's `Increase virtual disk size
+of a Clear Linux* OS image <https://clearlinux.org/documentation/clear-linux/
+guides/maintenance/increase-virtual-disk-size>`_ tutorial. The process can be
+broken down in three steps:
+
+1. Increase the virtual disk size
```

Comment:
```suggestion
1. Increase the virtual disk size defined in ``uos.img``
```

Reply:
I'm having mixed feelings about your suggestion here... the disk size isn't strictly defined in the `uos.img`, the file has a given size and contains partition that fill that space. How about the following instead?
```suggestion
1. Increase the virtual disk (``uos.img``) size
```